frontend/src/types/settings.d.ts

```diff
@@ -173,6 +173,13 @@ declare namespace Settings {
     excluded_tags: string[];
   }
 
+  interface Translating {
+    translator: string;
```

Comment:
maybe translator_type?

```suggestion
    translator_type: string;
```

Reply:
Done, thank you